internal/pkg/manifest/validate_env.go

```diff
@@ -318,12 +327,21 @@ func (al ELBAccessLogsArgs) validate() error {
 }
 
 // validate returns nil if ALBSecurityGroupsConfig is configured correctly.
-func (cfg ALBSecurityGroupsConfig) validate() error {
+func (cfg DeprecatedALBSecurityGroupsConfig) validate() error {
+	if cfg.Ingress.IsEmpty() {
+		return nil
+	}
```

Comment:
Sharing my thoughts:

This if-statement is skipping validating `Ingress` if it's empty. I think we should validate every single field no matter what. Whether `cfg.Ingress.IsEmpty` makes it a valid input should be `cfg.Ingress.validate()`'s call.

In terms of validation result, this doesn't make any difference here. I just wanted to share my thoughts on how I think about `validate`